chapters/object_types/samplers.txt

```diff
@@ -128,9 +139,10 @@ A three dimensional image sampler is created by calling <<object_types_sampler,
 | inOffset | `FLOAT32_VEC4` | (0, 0, 0, 0) | offset added to input transform result
 | image | `ARRAY3D` of <<Color>> |  | array backing the sampler
 | filter | `STRING` | `linear` | filter of the sampler, possible values: `nearest`, `linear`
-| wrapMode1 | `STRING` | `clampToEdge` | wrap mode of the sampler for the 1st dimension, possible values: `clampToEdge`, `repeat`, `mirrorRepeat`
-| wrapMode2 | `STRING` | `clampToEdge` | wrap mode of the sampler for the 2nd dimension, possible values: `clampToEdge`, `repeat`, `mirrorRepeat`
-| wrapMode3 | `STRING` | `clampToEdge` | wrap mode of the sampler for the 3rd dimension, possible values: `clampToEdge`, `repeat`, `mirrorRepeat`
+| wrapMode1 | `STRING` | `clampToEdge` | wrap mode of the sampler for the 1st dimension, possible values: `clampToBorder` (extension `KHR_SAMPLER_IMAGE_CLAMP_TO_BORDER`), `clampToEdge`, `repeat`, `mirrorRepeat`
+| wrapMode2 | `STRING` | `clampToEdge` | wrap mode of the sampler for the 2nd dimension, possible values: `clampToBorder` (extension `KHR_SAMPLER_IMAGE_CLAMP_TO_BORDER`), `clampToEdge`, `repeat`, `mirrorRepeat`
+| wrapMode3 | `STRING` | `clampToEdge` | wrap mode of the sampler for the 3rd dimension, possible values: `clampToBorder` (extension `KHR_SAMPLER_IMAGE_CLAMP_TO_BORDER`), `clampToEdge`, `repeat`, `mirrorRepeat`
+| borderColor | `FLOAT32_VEC4` | (0, 0, 0, 0) | border color used when wrapMode is `clampToBorder`
```

Comment:
Please add here as well in front
extension `KHR_SAMPLER_IMAGExD_CLAMP_TO_BORDER`, 
(similar the other places)